server/controllers/serviceMetrics/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from 'express'
+import data from '../../../fake-manage-recalls-api/stubs/__files/get-service-metrics.json'
+import { PhaseTimingAverages } from '../../@types/manage-recalls-api/models/PhaseTimingAverages'
+
+const getLabel = (name: string) => {
+  switch (name) {
+    case '7days':
```

Comment:
Dont believe the backend can have a variable starting with a number. Will confirm structure asap, but was looking at 'weekly'?

Reply:
'lastSevenDays'?
it's more specific than 'weekly'